repos/tap/theme/components/definitions/definitionList.js

```diff
@@ -177,6 +182,7 @@ const listItem = theme => {
         action: {
           touchable: {
             c: tapColors?.primary,
+            backgroundColor: '#fff',
```

Comment:
This line should be changed to: `bgC: theme?.colors?.palette?.white01`
* The `bgC` stands for backgroundColor.
* You can see a list of all alias [here](https://github.com/simpleviewinc/keg-hub/blob/develop/repos/re-theme/src/constants/ruleHelpers.js)

Reply:
Done. Thanks for the alias list